geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/WANTestBase.java

```diff
@@ -2302,6 +2302,25 @@ public static void doPuts(String regionName, int numPuts) {
     }
   }
 
+  public static void doTxPuts(String regionName, int numPuts) {
+    IgnoredException exp1 =
+        IgnoredException.addIgnoredException(InterruptedException.class.getName());
+    IgnoredException exp2 =
+        IgnoredException.addIgnoredException(GatewaySenderException.class.getName());
+    try {
+      Region<Object, Object> r = cache.getRegion(SEPARATOR + regionName);
+      assertNotNull(r);
+      for (long i = 0; i < numPuts; i++) {
+        cache.getCacheTransactionManager().begin();
+        r.put(i, "Value_" + i);
+        cache.getCacheTransactionManager().commit();
+      }
+    } finally {
+      exp1.remove();
+      exp2.remove();
+    }
+  }
+
```

Comment:
Another syntax which is a bit cleaner:
```
import static org.apache.geode.test.dunit.IgnoredException.addIgnoredException;

try (IgnoredException ie1 = addIgnoredException(InterruptedException.class);
       IgnoredException ie2 = addIgnoredException(GatewaySenderException.class)) {
  // ...
}
```
No need for a `finally` block when you do it as a try-with-resources like this.